manage_sections/views.py

```diff
@@ -276,8 +280,8 @@ def edit_section(request, section_id):
                   {'section': course_section})
 
 
-@login_required
-@lti_permission_required('manage_sections')
+# @login_required
+# @lti_permission_required('manage_sections')
```

Comment:
Should these be uncommented?

Reply:
Yes, I cleaned up a bunch of places. missed this!